src/plugins/lua/ratelimit.lua

```diff
@@ -560,14 +560,11 @@ if opts then
     -- new way of setting limits
     fun.each(function(t, lim)
       if type(lim) == 'table' then
-        if type(lim[2]) == 'number' then
-          settings[t] = lim
-        elseif type(lim[2]) == 'string' then
-          settings[t] = lim
-          local plim = parse_string_limit(lim[2])
-          if plim then
-            settings[t][2] = plim
-          end
+        settings[t] = lim
+      elseif type(lim) == 'string' then
+        local plim = parse_string_limit(lim)
+        if plim then
+          settings[t] = {0, plim, 1}
```

Comment:
The bucket size is 2 and leak rate is 2 / 60. So it will be empty in two minutes allowing 2 messages to be accepted.